src/libs/ReportUtils.js

```diff
@@ -524,12 +524,22 @@ function isPolicyExpenseChatAdmin(report, policies) {
 }
 
 /**
- * Returns true if report has a parent and is therefore a Thread.
+ * Returns true if report has a parent
  *
  * @param {Object} report
  * @returns {Boolean}
  */
 function isThread(report) {
```

Comment:
I think we could rename this to avoid confusion with `isChatThread`. Maybe `hasParentReport`?

Reply:
i personally like `isThread` cause it's simple. `hasParentReport` feels a little misleading to me cause we check for parentReportID and parentReportActionID. So then i would have to name it `hasParentReportIDAndActionID`, which feels unnecessary to me because that's what a "thread" is.

isChatThread - just checks if isThread and type of report is chat. I can easily see us expanding to do isIOUThread etc in future if needed too 🤷‍♂️

Let me know if you still feel differently about this.